common/handy/lists.go

```diff
@@ -0,0 +1,11 @@
+package handy
+
+type Lists[T any] map[string][]T
```

Comment:
General data structure.
In this PR we use this `handy.Lists` to conceptually split fields into required and optional kinds. Adding elements is convenient. Looping over values allows to apply different operations based on the list kind.